RoutingStrategies.xaml

```diff
@@ -0,0 +1,171 @@
+            <CollectionView x:Name="RoutesCollectionView" 
+                            HorizontalOptions="FillAndExpand"
+                            VerticalOptions="FillAndExpand"
+                            ItemsSource="{Binding Routes}">
+                <CollectionView.Header>
+                    <!-- Use Grid to ensure full width for columns -->
+                    <Grid ColumnDefinitions="1*, 2*, 1*" HorizontalOptions="FillAndExpand">
+                        <Label Text="ICAO" 
+                               FontAttributes="Bold" 
+                               HorizontalTextAlignment="Start"
+                               Grid.Column="0"
+                               Padding="0,0,10,0"/>
+                        <Label Text="City" 
+                               FontAttributes="Bold" 
+                               HorizontalTextAlignment="Start"
+                               Grid.Column="1"
+                               Padding="0,0,10,0"/>
+                        <Label Text="Dist" 
+                               FontAttributes="Bold" 
+                               HorizontalTextAlignment="End"
+                               Grid.Column="2"
+                               Padding="0,0,10,0"/>
+                    </Grid>
+                </CollectionView.Header>
+                
+                <!-- Data template for table rows -->
+                <CollectionView.ItemTemplate>
+                    <DataTemplate>
+                        <!-- Using Grid for row layout -->
+                        <Grid ColumnDefinitions="1*, 2*, 1*" Padding="0,10,0,0" HorizontalOptions="FillAndExpand">
+                            <Label Text="{Binding ICAO}" 
+                                   Grid.Column="0"
+                                   HorizontalTextAlignment="Start"
+                                   Padding="0,0,10,0"/>
+                            <Label Text="{Binding City}" 
+                                   Grid.Column="1"
+                                   HorizontalTextAlignment="Start"
+                                   Padding="0,0,10,0"/>
+                            <Label Text="{Binding Distance}" 
+                                   Grid.Column="2"
+                                   HorizontalTextAlignment="End"
+                                   Padding="0,0,10,0"/>
+                        </Grid>
+                    </DataTemplate>
+                </CollectionView.ItemTemplate>
+            </CollectionView>
+
+            <!-- Map takes up the rest of the space -->
+            <Label Text="Map" 
+                   FontAttributes="Bold" 
+                   FontSize="Medium" 
+                   Margin="0,20,0,10"/>
+            <maps:Map x:Name="map" 
+                      HorizontalOptions="FillAndExpand" 
+                      VerticalOptions="FillAndExpand"
+                      HeightRequest="300"/>
+        </VerticalStackLayout>
+    </ContentPage>
+
+    <!-- Additional Tabs -->
```

Comment:
Lines 149 to the end are also unnecessary and should be deleted - these pages are being created in MainTabbedPage, there's no need to recreate them here.